rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RealPreference.java

```diff
@@ -80,7 +79,9 @@
   }
 
   @Override public void delete() {
```

Comment:
add `synchronized` to `delete`, now, too.
(it exists for the `contains` and `get` calls in `RealPreference.get`.)